client/components/examples/activity/NewActivity.vue

```diff
@@ -9,9 +9,14 @@
         </el-col>
         <el-col :xs="24" :sm="{span: 10, offset: 2}">
           <el-form-item :label="$t('activity.area')" prop="region">
-            <el-select v-model="formData.region" :placeholder="$t('activity.holder.area')">
-              <el-option :label="$t('activity.city.sh')" value="shanghai"></el-option>
-              <el-option :label="$t('activity.city.bj')" value="beijing"></el-option>
+            <el-select v-model="formData.region" filterable allow-create :placeholder="$t('activity.holder.area')">
```

Comment:
Those were hardcoded, I needed to test out to understand if I got that part correctly.